opentech/apply/projects/views.py

```diff
@@ -120,12 +138,28 @@ def percentage(total, value):
 
             return rounded_total
 
+        def get_values(qs):
+            """
+            Get definitive values for each PaymentRequest
+
+            This finds the definitive value for each PaymentRequest, picking
+            `paid_value` when it has a value or `requested_value` when it does
+            not.  These annotated values are then returned as an iterable.
+            """
+            return qs.annotate(
```

Comment:
You should just be able to use Coalesce: `Coalesce(F('paid_value'), F('requested_value'))`

could also pull the sum in if you fancy...
```
qs.aggregate(
    total=Sum(Coalesce(F('paid_value'), F('requested_value')))
)['total']
```

Kind of tempted to move this into the queryset. (which you may be doing as part of the sprint 3 work on tables)

Reply:
Fixed and rebased.

I kept the logic in the inner function for now, we can pull it out later if there are more places which need it.